reconcile/utils/terrascript_aws_client.py

```diff
@@ -5469,9 +5469,13 @@ def populate_tf_resource_asg(self, spec) -> None:
                 override
             )
 
-        asg_value["tag"] = [
+        tags = [
             {"key": k, "value": v, "propagate_at_launch": True} for k, v in tags.items()
         ]
+        if self.versions.get(account, "").startswith("3"):
+            asg_value["tags"] = tags
```

Comment:
it seems asg 3.x also support `tag` and `tags` but it seems `tags` is something they call interpolated tags 

https://registry.terraform.io/providers/hashicorp/aws/3.73.0/docs/resources/autoscaling_group#interpolated-tags

Reply:
ah just saw the `No changes` idea. makes sense